src/main/java/com/mentalhealthapi/mentalhealthapi/dto/Disorder.java

```diff
@@ -6,6 +6,6 @@
     private int id;
     private String name;
     private String description;
-    private String dsm5Page;
+    private int dsm5Page;
```

Comment:
The dsm5Page variable could be set as int type since page numbers are positive integers. This would be more in line with Baeldung's characteristics of clean code. https://www.baeldung.com/java-clean-code#characteristics_of_clean_code